### ISSUE TITLE
Diagnosing Eady growth rates using the power method

### ISSUE THREAD

**Issue by glwagner** [glwagner/EadyTurbulence#6]
Oceananigans' Kelvin-Helmholz example:

https://clima.github.io/OceananigansDocumentation/stable/generated/kelvin_helmholtz_instability/

demonstrates how to use the power method to diagnose eigenvalues (growth rates) and eigenmodes of a linear instability problem in Oceananigans. We should do the same for an Eady instability problem.

Might make sense to modify `equilibrated/run_small_eady_problem.jl` for this purpose and start a new directory `power_method` or something.

**Comment by navidcy** [comment]
Yes. Is `equilibrated/run_small_eady_problem.jl` the best place for me to start?

**Comment by glwagner** [comment]
Might make sense to modify equilibrated/run_small_eady_problem.jl for this purpose and start a new directory power_method or something.

**Comment by navidcy** [comment]
ok, I will do that tomorrow perhaps...
